src/Resources/views/storefront/components/factfinder/record-list.html.twig

```diff
@@ -2,7 +2,9 @@
   <ff-record-list
     {% if id %} id="{{ id }}" {% endif %}
     class="{{ class }}"
-    subscribe="{{ subscribe ? 'true' : 'false' }}" {{ infiniteScrolling ? 'infinite-scrolling' : '' }} {{ infiniteScrolling ? 'infinite-debounce-delay="infiniteDebounceDelay"' : '' }}
+    subscribe="{{ subscribe ? 'true' : 'false' }}"
+    {{ infiniteScrolling ? 'infinite-scrolling' : '' }}
+    {{ infiniteScrolling ? "infinite-debounce-delay=#{infiniteDebounceDelay}" : '' }}
```

Comment:
Can you put these 2 conditions in one line?

Reply:
like this ?
![image](https://user-images.githubusercontent.com/4511883/135414487-1589d13e-9b60-426d-9233-6fd66d58e198.png)

Reply:
maybe something like this `{{ infiniteScrolling ? 'infinite-scrolling infinite-debounce-delay=#{infiniteDebounceDelay}' : '' }}` ?

Reply:
yeah, that will be better, changed